File: p0/Solution.java
import java.io.*;

public class Solution {
    public static void main(String[] args) throws IOException {
        BufferedReader bufferedReader = new BufferedReader(new InputStreamReader(System.in));

        String[] firstMultipleInput = bufferedReader.readLine().replaceAll("\\s+$", "").split(" ");

        int n = Integer.parseInt(firstMultipleInput[0]);
        int m = Integer.parseInt(firstMultipleInput[1]);

        String[] productivitiesTemp = bufferedReader.readLine().replaceAll("\\s+$", "").split(" ");
        bufferedReader.close();
      
        int[] productivities = new int[n]; // Technically unnecessary (why?), but it's ok to keep
        for (int i = 0; i < n; i++) {
            productivities[i] = Integer.parseInt(productivitiesTemp[i]);
        }

        int diff = 0; // The total differential in productivity we can attain
        for (int p : productivities) {
            if (m > p) diff += m - p;
        }

        System.out.println(diff);
    }
}

File: p0/solution.cpp
#include <iostream>
using namespace std;

int main() {
    int n, m, current;
  
    cin >> n >> m;

    int diff = 0; // The total differential in productivity we can attain
    for (int i = 0; i < n; i++) {
        cin >> current;
        diff += (m > current) ? m - current : 0;
    }
  
    cout << diff << endl;

    return 0;
}